.goreleaser.yml

```diff
@@ -1,33 +1,61 @@
 # You can find the GoReleaser documentation at http://goreleaser.com
 project_name: exercism
 
+env:
+  - CGO_ENABLED=0
 builds:
-- env:
-    - CGO_ENABLED=0
-  main: ./exercism/main.go
-  goos:
-    - darwin
-    - linux
-    - windows
-    - freebsd
-    - openbsd
-  goarch:
-    - amd64
-    - 386
-    - arm
-    - arm64
-    - ppc64
-  goarm:
-    - 5
-    - 6
-  ignore:
-    - goos: openbsd
-      goarch: arm
-    - goos: freebsd
-      goarch: arm
+  - id: release-build
+    main: ./exercism/main.go
+    mod_timestamp: '{{ .CommitTimestamp  }}'
```

Comment:
What does this do?

Reply:
The mod_timestamp configuration modifies the creation time of the built binary to match the value assigned to it. In this case, the creation time will be set to the time of the last commit within the tagged release.  If left unset it would just be the compilation time. 

This is done to ensure running the same job over results in reproducible (go binaries) and checksums. https://goreleaser.com/customization/builds/#reproducible-builds